src/app/ngrx/app.reducer.ts

```diff
@@ -0,0 +1,308 @@
+import { createReducer, on } from '@ngrx/store';
+import { AppState } from '../model/appstate.model';
+import { addArtist, addInitialData } from './app.action';
+
+export const initialState: AppState = {
+    artists: [
```

Comment:
It would be great if you could separate the data another file, like the mocks that we have in the it project. And import here. It is little bothersome to scroll down

Reply:
ok, but i will do that in another story where i will syncronize the local storage and the store ok? this way you can start using the store sooner

Reply:
oki